llpc/unittests/util/testPipelineDumper.cpp

```diff
@@ -291,8 +291,22 @@ TEST(PipelineDumperTest, TestForceCsThreadIdSwizzlingCompute) {
   HashModifiedFunc expectHashToBeEqual = [](const GenerateHashParams &params) { return false; };
   runComputePipelineVariations(modifyBuildInfo, expectHashToBeEqual);
 }
-
 #endif
 
+// =====================================================================================================================
+// Test the threadGroupSwizzleMode option.
+
+TEST(PipelineDumperTest, TestThreadGroupSwizzleModeCompute) {
+  for (uint32_t threadGroupSwizzleMode = static_cast<uint32_t>(ThreadGroupSwizzleMode::_4x4);
+       threadGroupSwizzleMode < static_cast<uint32_t>(ThreadGroupSwizzleMode::Count); ++threadGroupSwizzleMode) {
```

Comment:
Please use `lgc::enumRange` to iterate over enums: https://github.com/GPUOpen-Drivers/llpc/blob/dev/lgc/interface/lgc/EnumIterator.h#L128

Also, why do we use `uint32_t` here instead of `unsigned` like before? If you just want the integer type to match the enum type, you can use `toUnderlying`: https://github.com/GPUOpen-Drivers/llpc/blob/dev/lgc/interface/lgc/EnumIterator.h#L81

Reply:
I use `uint32_t` just because it is used elsewhere in this file...